packages/status-react/src/styles/config.tsx

```diff
@@ -161,6 +161,182 @@ export const darkTheme = createTheme({
   },
 })
 
+// todo: check source licenses
+/**
+ * Base styles.
+ *
+ * @see https://tailwindcss.com/docs/preflight for styles source
+ * @see https://unpkg.com/tailwindcss@3.1.8/src/css/preflight.css for styles source
+ * @see https://github.com/codesandbox/sandpack/blob/main/sandpack-react/src/contexts/themeContext.tsx for styles source
+ * @see https://caniuse.com/?search=%3Awhere() for browser support
+ *
+ * note: Check regurarly for changes in the upstreams.
```

Comment:
This changes very rarely.

Reply:
https://github.com/status-im/status-web/pull/314#issuecomment-1268364751